public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilderOptions.tsx

```diff
@@ -19,12 +20,15 @@ export interface Props {
   maxLines: number;
   app?: CoreApp;
   datasource: LokiDatasource;
+  queryStats: QueryStats | undefined;
+  setQueryStats: React.Dispatch<React.SetStateAction<QueryStats | undefined>>;
```

Comment:
I am thinking if it wouldn't be better if we run `setQueryStats` in `LokiQueryField` and used this component just to display stats.